index.html

```diff
@@ -4767,7 +4819,9 @@ <h3>Further Information</h3>
       <p>
 The W3C Internationalisation document [[string-meta]] gives more information
```

Comment:
I'd prefer an early citation to a late one?

Reply:
Done. We mention it at the top now: b2f3fdb952a9ca4a96411170bb5d465334cdcad3